source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -348,14 +520,9 @@ Related References:
 }
 ```
 
-To create an app with App Components, you will need to create an OAuth App and request the 
-App Components functionality onto it via the 
-[Create an App Components App](https://form-beta.asana.com?k=LBWpDpqZ6b-6pV4ZIbP-OA&d=15793206719) form.
+The following table lists the configurations you can make to define your App Components app.
 
-To create an OAuth app, see [Authentication Quick Start](/docs/authentication-quick-start).
-
-To create an App Components app you'll need to fill out the [Create an App Components App](https://form-beta.asana.com?k=LBWpDpqZ6b-6pV4ZIbP-OA&d=15793206719)
-form with the data in the table below.
+To create an App Components app, you'll need to first [create an OAuth app](docs/authentication-quick-start), then fill out the [App Components Configuration form](https://form-beta.asana.com?k=LBWpDpqZ6b-6pV4ZIbP-OA&d=15793206719) with the data described in the table below.
```

Comment:
I think we should drop the phrase "Oauth app" from our documentation here (and in other places throughout the docs) since we've unified the concepts of "oauth app" and "app components app". So this could read something along the lines of

```
To create and app with App Components functionality, you'll first need to create an [app](link)...
```

Also, it might be worth calling out that certain fields in the developer console (namely the app name and icon) will also be used by the App Components system.

Reply:
Sounds good. I've made the changes and included the note on dev console customization.